code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -11,6 +11,8 @@
 			new /obj/item/device/camera_bug(src)
 			new /obj/item/device/multitool/ai_detect(src)
 			new /obj/item/device/encryptionkey/syndicate(src)
+			new /obj/item/weapon/reagent_containers/syringe/mulligan(src)
+			new /obj/item/weapon/switchblade

```

Comment:
needs a `(src)` here
also aren't 9 items a bit too much, do these even fit in a single box